config/database.yml

```diff
@@ -2,8 +2,8 @@ default: &default
   adapter: mysql2
   encoding: utf8
   username: root
-  password: 
-  host: 127.0.0.1
+  password:
+  host: <%= ENV['MYSQL_HOST'] || "127.0.0.1" %>
```

Comment:
variable MYSQL_HOST will be assigned different values depending on the environment, this is how it was done in original config file, should we keep it the same? This is similar to the way redis server is set. (taken from hyku config for redis)